CONTRIBUTING.md

```diff
@@ -23,6 +23,24 @@ This is a rough outline of what a contributor's workflow looks like:
 
 Thanks for your contributions!
 
+## Review Flow
+
+TiKV uses bors, a bot used by projects like Rust and Servo. If you are already familiar with how to use bors, please just use it.
+
+Reviewers should follow this process:
+
+- Find a PR to review, or be requested to review one.
+- Review it. Give complete feedback, make sure the contributor can understand your suggestions. If needed, help them.
+- If you approve it, use the 'Submit Review' button and choose 'Approve.'
+- If you are the first approver, request someone else to review it as well.
+- If you are the second approver, you can leave `bors r+` in the review or as a comment.
+    + If the change is trivial, or you are sure it cannot fail, you can use `bors rollup` to allow bors to batch a merge.
+- bors will merge the code after a time so long as there are no merge conflicts with master, and the tests pass.
+    + **You do not need to hit the 'Update Branch' button unless there are conflicts or you think it will fail.**
+- You will recieve a notification for the PR saying it was merged, or it failed.
+
+If you need support using bors you should consult [the guide](https://bors.tech/documentation/getting-started/), you may also ping @hoverbear.
```

Comment:
Is this `@` gonna work as a link?

Reply:
I'm not entirely sure, I'll make it an explicit link.